pysat/_orbits.py

```diff
@@ -649,17 +661,24 @@ def _orbit_number_breaks(self):
         else:
             ind = np.array([0])
 
-        # Set the index of orbit breaks and the number of orbits
+        # Set the index of orbit breaks and the number of orbits.
         self._orbit_breaks = ind
         self.num = len(ind)
 
+        return
+
     def _get_basic_orbit(self, orbit_num):
         """Load a particular orbit into .data for loaded day.
 
         Parameters
         ----------
         orbit_num : int
-            orbit number, 1 indexed, negative indexes allowed, -1 last orbit
+            Orbit number, 1 indexed, negative indexes allowed, -1 last orbit.
```

Comment:
I'm bringing up the 1 index since python standard is zero indexing and the type is covered by the `orbit_num : int` portion. I did update the comment though so hopefully things are more clear.